Resources/Textures/Clothing/Head/Hats/beret_cmo.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by 𝘽𝙚𝙡𝙖𝙮#7441",
```

Comment:
If this is a custom sprite, either get the spriter on SS14 #art-and-ui discord channel to confirm this or use their github username

Reply:
DNM until attribution is fixed